packages/core/__tests__/src/utils/framework.test.ts

```diff
@@ -16,10 +16,7 @@ describe("framework.detectFramework", () => {
     mockedFs.lstatSync.mockReturnValue({ dev: 1 } as fs.Stats);
     mockedFs.readFileSync.mockReturnValue(JSON.stringify({ dependencies: {} }));
```

Comment:
I think we can split a little bit this test into test for the detectFramework functions and tests for the defineFrameworks functions, and use a different describe for them